Makefile

```diff
@@ -243,22 +247,23 @@ container: .container-$(ARCH)-$(VERSION) container-name
 	@docker images -q $(IMAGE):$(ARCH)-$(VERSION) > $@
 
 container-latest: .container-$(ARCH)-$(VERSION)
-	@docker tag $(IMAGE):$(ARCH)-$(VERSION) $(IMAGE):$(ARCH)-latest
+	@docker tag $(IMAGE):$(ARCH)-$(VERSION) $(REGISTRY_REFIX)$(IMAGE):$(ARCH)-latest
 	@echo "container: $(IMAGE):$(ARCH)-latest"
 
 container-name:
 	@echo "container: $(IMAGE):$(ARCH)-$(VERSION)"
 
 manifest: .manifest-$(VERSION) manifest-name
 .manifest-$(VERSION): Dockerfile $(addprefix push-, $(ALL_ARCH))
-	@docker manifest create --amend $(IMAGE):$(VERSION) $(addsuffix -$(VERSION), $(addprefix squat/$(PROJECT):, $(ALL_ARCH)))
+	@docker manifest create --amend $(MANIFEST_IMAGE):$(VERSION) $(addsuffix -$(VERSION), $(addprefix $(MANIFEST_IMAGE):, $(ALL_ARCH)))
 	@$(MAKE) --no-print-directory manifest-annotate-$(VERSION)
-	@docker manifest push $(IMAGE):$(VERSION) > $@
+	@docker manifest push $(MANIFEST_IMAGE):$(VERSION) > $@
 
 manifest-latest: Dockerfile $(addprefix push-latest-, $(ALL_ARCH))
-	@docker manifest create --amend $(IMAGE):latest $(addsuffix -latest, $(addprefix squat/$(PROJECT):, $(ALL_ARCH)))
+	@docker manifest rm $(MANIFEST_IMAGE):latest || echo no old manifest
```

Comment:
Why do we need to rm the manifest first? Does it not just get overwritten? This seems to work automatically for kilo:latest today, but maybe this is special in docker hub?

Reply:
I was bitten several time by the manifest not being updated on our internal registry, don't know where it comes from, but removing it fixed the issue. Maybe it's only me :man_shrugging: